client/src/components/list/CreateUserList.tsx

```diff
@@ -0,0 +1,72 @@
+import { 
+    Input, 
+    Radio, 
+    Stack,
+    VStack,
+    Button,
+    Select,
+    Heading,
+    FormControl, 
+    FormLabel,
+    FormErrorMessage,
+    FormHelperText,
+    useDisclosure,
+    Center
+} from "@chakra-ui/react";
+
+import * as React from 'react';
+import{ Formik, Form, Field, useFormik } from "formik";
+import * as Yup from 'yup';
+import ChooseRatingModal from "./ChooseRatingModal";
+//import { RatingSystem } from "../../generated/graphql";
+
+
+const CreateUserList: React.FC<{}> = () => {
+
+    const { isOpen, onOpen, onClose } = useDisclosure()
+
+    const validationSchema = Yup.object({
+      name: Yup.string()
+        .min(1, 'Name must be at least 1 character long')
+        .max(50, 'Name must be at most 50 characters long')
+        .required('Required'),
+    });
+
+    const formik = useFormik({
+      initialValues:{
+        name: "",
+        ratingSystemID: ""
+      },
+      validationSchema,
+      onSubmit: (values, actions) => {
+        actions.setSubmitting(true);
+
+      }
+    });
+    return(
+        <form onSubmit={formik.handleSubmit}>
+          <Stack spacing={{ base: 8, md: 12 }} mt={4}>
+          <FormControl isRequired>
+            <FormLabel>Name</FormLabel>
+            <Input placeholder="MyUserList" id="name"/>
```

Comment:
For form inputs make sure to have an `id` prop as well as destructure the `formik.getFieldProps` for the appropriate input

Example: https://github.com/ucsb-cs148-f21/project-t10-animelist/blob/b2c9119256a6d3d3b11459c3bdc3463a0a763a14/client/src/components/forms/LoginForm.tsx#L68